data/virt_autotest/virt_logs_collector.sh

```diff
@@ -105,7 +105,7 @@ while { \${retry_times} > 0 } {
          expect -re "~( |\\\])#"
          if { \${extra_logs} == {support_config} } {
             send "rm -f -r \${logs_folder}/*supportconfig*\r"
-            send "supportconfig -y -A -o AUDIT -t \${logs_folder} -B guest_\${guest_transformed}_supportconfig_\\\${time_stamp}\r"
+            send "supportconfig -y -A -x aFSLIST,AUDIT -t \${logs_folder} -B guest_\${guest_transformed}_supportconfig_\\\${time_stamp}\r"
```

Comment:
It is okay to workaround them now, but it would be good to open a bug for them. Leon opened a bug for supportconfig earlier and dev has solutions to resolve those long time taking collections.

Reply:
I saw that bug. they add a timeout to supportconfig, but I guess the timeout is 1 hour, it made our tests one hour longer, and in some cases, it broke our tests. I think it is unacceptable.

The "File system list" excluded, it will generate filelist.txt to list the all the files in the system. some info can be querried from other txt files. It is possible it is not very necessary for virt bugs.

For opening a bug, it would be a better choice. it will take quite a few days to locate the root or fix it finally. It is pretty good that someone have this time slot and have it done. Afterword we can re-add the module to supportconfig files.

Reply:
I created a poo ticket to tack this, https://progress.opensuse.org/issues/152635

Reply:
Hi Julie,
the bug's fix is in https://build.opensuse.org/request/show/1127373. I do not find the timeout 1 hour. There is only one `find -mtime 30`, but it is to find the files within modified time within 30.

I am not sure if we are on the same page for "locate the root or fix it finally". My understanding is that when we open a bug for this, we do not need to find the root cause, and we only need to give the reproduce steps and the logs for dev to analyze. It's dev's job to fix it. Actually, for this problem, I guess it is similar with Leon's bug, the root cause is that the things it want to upload is not available. So the root cause is NOT in supportconfig tool, but they can make some improvement for it to NOT wait so long.

Reply:
then I guess Leon's bug is still applicable with this issue. This issue is not a new one, it is the same with Leon's bug. We need not open a new bug.

Reply:
@Julie-CAO  Okay, then let's merge the code now. Please help to update the bug accordingly.